mindsdb/integrations/handlers/merlion_handler/adapters.py

```diff
@@ -0,0 +1,182 @@
+from merlion.models.anomaly.forecast_based.prophet import ProphetDetectorConfig, ProphetDetector
+from merlion.models.anomaly.isolation_forest import IsolationForestConfig, IsolationForest
+from merlion.models.anomaly.windstats import WindStatsConfig, WindStats
+from merlion.models.automl.autoprophet import AutoProphetConfig, AutoProphet
+from merlion.models.automl.autosarima import AutoSarimaConfig, AutoSarima
+from merlion.models.forecast.smoother import MSESConfig, MSES
+from merlion.post_process.threshold import AggregateAlarms
+from merlion.transform.moving_average import DifferenceTransform
+from scipy.stats import norm
+
+from merlion.models.defaults import DefaultDetectorConfig, DefaultDetector, DefaultForecaster, DefaultForecasterConfig
+from merlion.utils import TimeSeries
+
+from enum import Enum
+import pandas as pd
+
+
+class MerlionArguments(Enum):
+    target_seq_index = "target_seq_index"
+    max_forecast_steps = "max_forecast_steps"
+    max_backstep = "max_backstep"
+    wind_sz = "wind_sz"
+    alm_threshold = "alm_threshold"
+    maxiter = "maxiter"
+
+
+class BaseMerlionForecastAdapter:
+    TARGET_SEQ_INDEX = 0
+    DEFAULT_MAX_FORECAST_STEPS = 100
+    DEFAULT_MAX_BACKSTEP = 60
+    DEFAULT_MAXITER = 5
+
+    def __init__(self, **kwargs):
+        self.max_forecast_steps = kwargs.get(MerlionArguments.max_forecast_steps.value, self.DEFAULT_MAX_FORECAST_STEPS)
+        self.max_backstep = kwargs.get(MerlionArguments.max_backstep.value, self.DEFAULT_MAX_BACKSTEP)
+        self.maxiter = kwargs.get(MerlionArguments.maxiter.value, self.DEFAULT_MAXITER)
+        self.model = None
+
+    def to_bytes(self):
+        return self.model.to_bytes()
+
+    def from_bytes(self, bytes):
```

Comment:
`to_bytes` and `from_bytes` should be consistent in signature. My vote is for `return`ing `self.model` in `from_bytes()`

Reply:
ok, to_bytes will have the return signature bytes,

from_bytes has been renamed to initialize_model, I think this is what the function do.